cumulus/pallets/xcmp-queue/src/tests.rs

```diff
@@ -422,19 +424,23 @@ fn verify_fee_factor_increase_and_decrease() {
 		assert_ok!(send_xcm::<XcmpQueue>(destination, xcm.clone())); // Size 520
 		assert_eq!(DeliveryFeeFactor::<Test>::get(sibling_para_id), FixedU128::from_float(1.05));
 
-		for _ in 0..12 { // We finish at size 929
+		for _ in 0..12 {
+			// We finish at size 929
```

Comment:
With what was this formatted?  
Normally its a good idea to minimize the diff to quicken the review/audit requirements.

Reply:
I had the same comment [here](https://github.com/paritytech/polkadot-sdk/pull/2007#pullrequestreview-1695269783). The formatting is correct, but I would keep it in a separate PR

Reply:
I ran the command `cargo +nightly fmt --all`, I suppose I should run it without the `--all` flag.

Reply:
The CI is pinned to a very specific version: `cargo +nightly-2023-05-16 fmt` is what i use currently.

Reply:
Just ran `cargo +nightly-2023-05-16 fmt` command and no changes. I pulled in the errant fmt files from master so the diff is correct now.